src/NHSDPublicBrowseAcceptanceTests.Tests/Utils/EnvironmentVariables.cs

```diff
@@ -60,8 +69,23 @@ private static string JsonConfigValues(string section, string defaultValue)
 
             return string.IsNullOrEmpty(result) ? defaultValue : result;
         }
-    }
 
+        private static string DefaultUri()
+        {
+            var uri = "https://docker.for.win.localhost";
+            //if (Debugger.IsAttached)
+            //{
+                //var builder = new UriBuilder(uri)
+                //{
+                //    Host = "localhost"
+                //};
+
+                //uri = builder.Uri.ToString();
+            //}
```

Comment:
is this debugger localhost no longer needed?

Reply:
I forgot I had commented out, will be removing the comments shortly